include/devicetree/spi.h

```diff
@@ -27,15 +27,15 @@ extern "C" {
  * @param spi node identifier for a SPI bus controller
  * @return 1 if it has a cs-gpios property, 0 otherwise
  */
-#define DT_SPI_HAS_CS(spi) DT_NODE_HAS_PROP(spi, cs_gpios)
+#define DT_SPI_HAS_CS_GPIO(spi) DT_NODE_HAS_PROP(spi, cs_gpios)
```

Comment:
wouldn't it be more consistent to use GPIOS (plural)?

Reply:
I agree.  `DT_SPI_HAS_CS_GPIO` suggests to me it would be testing whether the SPI peripheral uses a GPIO-based CS versus a `reg`-based one.

Reply:
ok, renamed all the macros to use `CS_GPIOS`